import_package.md

```diff
@@ -0,0 +1,281 @@
+# Import package
+
+An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+A valid import package must contain all resource files that are referenced in the 
+  **manifest.yaml**.
+
+## Meta-data
+
+* `apiVersion` is set to `v1beta1` 
+
+## Tasks
+
+* `tasks`: list of api actions and resource tasks.
+    * `id`:  unique identifier of the task.  Must contain only upper-case, 
+       lower-case letters, digits or underscores (`[A-Za-z0-9_]+`)
+    * `type`: Must be either **api** or **resource**. All required properties of the specified
+       type have to be implemented.
+    * `name` *(optional)*: A human readable name for the action to easier identify the action
+    * `context` *(optional)*: An optional context which can contain additional data for the 
+      API action or resource task. A context must only contain string values and supports templating.
+
+### API action
+An API action is a task that interacts with a specific Keptn API endpoint. The JSON results of the endpoint
+are stored in ` .Tasks.<task-id>.Response.<field>` and are accessible for all following tasks. The following
+properties have to be implemented by an API action:
+* `action`: The name of the Keptn API endpoint to call. 
+  See [Supported API endpoints](#supported-api-endpoints) for more information.
+* `payload`: A reference to a file that should be used as body payload for the API action.
+  If the action does not take a body, this property is ignored. The file reference has to be
+  specified as the full path within the provided import package.
+
+### Resource task
+A resource task adds resources to Keptn. Similar to the API actions, the resources are also 
+processed by the templating engine. The following properties have to be defined for a resource task:
+* `resource`: A reference (file path) to a file in the import package
+* `resourceUri`: The URI at which the file should be stored in Keptn
+* `stage` *(optional)*: An optional stage name that can be specified
+* `service` *(optional)*: An optional service that can be specified, if specified `stage` has to be non-empty
+
```

Comment:
Maybe we can add that the resource tasks works similar to `keptn add-resource` and reference the docs (e.g., we didn't invent the parameters resource and resourceUri): https://keptn.sh/docs/0.18.x/reference/cli/commands/keptn_add-resource/